EasyVCR/Censors.cs

```diff
@@ -34,9 +33,9 @@ public static Censors DefaultSensitive
             get
             {
                 var censors = new Censors();
-                censors.HideHeaders(Defaults.CredentialHeadersToHide);
-                censors.HideQueryParameters(Defaults.CredentialParametersToHide);
-                censors.HideBodyParameters(Defaults.CredentialParametersToHide);
+                censors.HideHeaderKeys(Defaults.CredentialHeadersToHide);
```

Comment:
Do these indeed hide the key itself or the data of the key? If it's the data of the key, I'd recommend against this naming convention as it could be misleading. To me, this says we are only going to hide the key and not the value associated with it.

I'd also suggest a different word than `hide` since that implies it can still be found when in reality it cannot once censored. Honestly `Censor` is probably the best option?